Makefile

```diff
@@ -831,14 +831,18 @@ ifeq "$(OPENMP_OFFLOAD)" "true"
 	LDFLAGS += $(LDFLAGS_GPU)
 endif #OPENMP_OFFLOAD IF
 
-ifeq "$(PRECISION)" "single"
+ifneq (,$(filter-out double single,$(PRECISION)))
+$(error PRECISION should be "", "single", or "double"; recieved value "$(PRECISION)")
```

Comment:
`recieved` -> `received`

Reply:
> Although the HEAD of the PR branch does compile, a261cdbc doesn't compile if PRECISION=single or PRECISION is not specified. Generally, I think it's good to have every commit leave the code in a state that compiles (and ideally, runs).

Whoops! It seems that I rebased poorly and added that fix to 3f25e68 instead of where I meant to. If you don't mind, I can do another force push to fix that and the mispelling of `received`.